### PR TITLE
Parameters upadate

### PR DESCRIPTION
**Pull request by danilodileo**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/metatdenovo/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/metatdenovo _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: nextflow_schema.json [hunk 1]
```diff
@@ -107,35 +107,6 @@
...
-                    "description": "Comma-separated list of hmm files which will be searched for among ORFs",
-                    "pattern": "\\S+hmm(\\.gz)?",
-                    "format": "file-path",
-                    "mimetype": "text/plain",
-                    "fa_icon": "far fa-file-code"
-                }
-            }
-        },
         "bbduk_options": {
             "title": "BBDUK options",
```
Comment (erikrikarddaniel) [inline nextflow_schema.json]:
```suggestion
            "title": "BBDuk contamination removal options",
```

Comment (erikrikarddaniel) [reply nextflow_schema.json]: Maybe we should be more explicit in other places too?

File: nextflow_schema.json [hunk 2]
```diff
@@ -107,35 +107,6 @@
...
-                    "pattern": "^\\S+",
-                    "format": "directory-path",
-                    "fa_icon": "fas fa-folder-open"
-                },
-                "hmmpattern": {
-                    "type": "string",
-                    "default": "*.hmm",
-                    "description": "specify which pattern hmm files end with",
-                    "fa_icon": "fas fa-barcode"
-                },
```
Comment (erikrikarddaniel) [inline nextflow_schema.json:128]: Where did this go, don't we use it?

Comment (erikrikarddaniel) [reply nextflow_schema.json:128]: Now I see, you put it among the hidden options. Is that how similar options are handled by other pipelines?

**Review by erikrikarddaniel — APPROVED** [review]
A few comments. I'm not 100% that the order is the best, but fine for now. I'm approving, but check the  comments.

File: nextflow_schema.json (hunk 2 above)
Comment (danilodileo) [reply nextflow_schema.json:128]: I think that it is a mistake, it should be visible

**Pull request merged**